app/controllers/assessments_controller.rb

```diff
@@ -61,10 +58,20 @@ def type_param
   end
 
   def assessment_params
-    params.require(:need).permit(:user_id, :role_id, :name, :is_urgent, :status, :category, :status, :start_on)
+    params.require(:need).permit(:assigned_to, :name, :is_urgent, :status, :category, :status, :start_on)
   end
 
   def notes_params
     params.require(:note).permit(:body)
   end
+
+  def construct_assigned_to_options
```

Comment:
This is duplicated in both controllers.
Also wondering if this is an option for caching (ok premature optimisation, but certainly an option for the future once someone has done some performance tests)